rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1598,108 +1598,6 @@ def process_contamination_category_locally(self):
 
         return 0
 
-
-class IngestionClass(GenomicManifestBase):
```

Comment:
Why is IngestionClass being removed?

Reply:
I am trying too clean up tools that aren't in use anymore, do you still use this?

Reply:
Thanks, I use this tool when running AW1 reingestions.